proposals/LP0004-project-governance.md

```diff
@@ -0,0 +1,448 @@
+
+We give _respect freely_, but _authority is earned_. Since we are all peers, everyone in the community deserves and should be given respect. Authority is earned through merit and is fluid; we favor current active participants and we prioritize creating opportunities for newcomers to gain influence.
+
+We expect _public communication_. As an increasingly large and dispersed organization we rely more and more on public and asynchronous communication.
+
+We value _community over code_. We believe that a healthy and strong community can overcome any obstacle, code or otherwise. For this reason we will never sacrifice the health of our community for a technical contribution.
+
+We embrace a _broad definition of contribution_. Throughout this document we talk about contributions and community involvement. The definition of _contribution_ is not limited to just code commits. We value and recognize the many shapes that _contributions_ take.
+
+### Consensus-Seeking Decision Making
+
+The spirit of [LP0001 LLVM Decision Making](https://github.com/llvm/llvm-www/blob/main/proposals/LP0001-LLVMDecisionMaking.md) is to provide a framework to help the community arrive at a decision around contentious topics. As described the process does not follow a fully democratic process and it acknowledges the possibility that consensus cannot be reached. These are strengths of that proposal process and nothing in this proposal is intended to weaken that.
+
+Formal consensus and majority opinion (i.e. voting, surveys, etc) processes are _extremely_ difficult to execute on decentralized projects like LLVM, and not necessary for healthy community growth. Formal consensus or majority opinion are not goals.
+
+Consensus-seeking decision making instead focuses on efforts to build consensus and address concerns. It does not require that all concerns are addressed, nor that a full consensus is reached. It also does not require a majority approval.
+
+This proposal gives the responsibility for facilitating consensus-seeking and last-resort decision making authority to project governance structures defined below.
+
+
+### Role of Code Owners
+
+This proposal does not change the role of code owners as defined in the [LLVM Developer Policy](https://llvm.org/docs/DeveloperPolicy.html#code-owners). As stated in the policy:
+
+    The sole responsibility of a code owner is to ensure that a commit to their area of the code is appropriately reviewed, either by themself or by someone else.
+
+
+### Code of Conduct
+
+All participants in the LLVM Community must follow the LLVM Community Code of Conduct. That applies to the representatives of governance bodies just as it does to any other individual in the community. The Code of Conduct Committee can suspend or revoke the privilege of any individual to participate in the LLVM Governance process for violations of the Code of Conduct.
```

Comment:
s/the LLVM Community Code of Conduct/[the LLVM Community Code of Conduct](https://llvm.org/docs/CodeOfConduct.html)/

s/Code of Conduct Committee/[Code of Conduct Committee](https://llvm.org/docs/CodeOfConduct.html#code-of-conduct-committee)/

(add links)